release/version

```diff
@@ -1 +1 @@
-v1.3.1
+v1.3.1-patch
```

Comment:
can we say `1.3.1-patchOBS` to be explicit.

Reply:
rather, 1.3.1-patchobs-<bugzilla>

Reply:
Updated with the 1.3.1-patchobs-bugid, thanks Arun!